src/3d/qgs3dmapscene.cpp

```diff
@@ -247,6 +247,36 @@ void Qgs3DMapScene::viewZoomFull()
   mCameraController->resetView( 1.5 * std::sqrt( a * a - side * side ) );  // assuming FOV being 45 degrees
 }
 
+void Qgs3DMapScene::viewExtent( const QgsRectangle &extent )
```

Comment:
The naming is a bit confusing as it implies it returns extent of the view, but in fact it sets the camera, so how about something like `setViewFrom2DExtent()` ?